### PR TITLE
Migrate more images

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: doi.jq [hunk 1]
```diff
@@ -54,10 +53,8 @@ def repos_anti_subset:
 		"mediawiki",
 		"monica",
 		"nextcloud",
-		"python", # direct children: hylang plone satosa unit
```
Comment (whalelines) [inline doi.jq:57]: No SBOM for python?

**Review by whalelines — COMMENTED** [review]
Question about python

File: doi.jq (hunk 1 above)
Comment (LaurentGoderre) [reply doi.jq:57]: Python was accidentally added to the SBOM list is seems but it wouldn't get them without being on the new build

Comment (LaurentGoderre) [reply doi.jq:57]: AAaaah, good point. It never occured to me to do it that way

**Review by tianon — APPROVED** [review]

**Pull request merged**